bridge/xmpp/helpers.go

```diff
@@ -0,0 +1,29 @@
+package bxmpp
+
+import (
+	"regexp"
+
+	"github.com/42wim/matterbridge/bridge/config"
+)
+
+// GetAvatar constructs a URL for a given user-avatar if it is available in the cache.
+func getAvatar(av map[string]string, userid string, general *config.Protocol) string {
+	if hash, ok := av[userid]; ok {
+		// NOTE: This does not happen in bridge/helper/helper.go but messes up XMPP
+		reg := regexp.MustCompile("[^a-zA-Z0-9]+")
```

Comment:
You can probably compile this at the start of the function or at the package level

Reply:
I just pushed that change